lesson13/style.css

```diff
@@ -1,20 +1,134 @@
-.loading {
-  width: 100vw;
-  height: 100vh;
-  position: fixed;
-  top: 0;
-  left: 0;
-  z-index: 9999;
+
+/* reset css
+=====================================================*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
 }
 
-.loaded {
-  opacity: 0;
-  visibility: hidden;
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+
+body {
+	line-height: 1;
 }
 
+ol, ul {
+	list-style: none;
+}
+
+blockquote, q {
+	quotes: none;
+}
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* =====================================================*/
+
+/* common parts */
 .button {
+  cursor: pointer;
   display: block;
   background-color: white;
   border: 2px solid black;
   padding: 10px 30px;
 }
+
+.hidden {
+  display: none;
+}
+
+/* loading img */
+.loading {
+  width: 50%;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.loading img {
+  width: 50%;
+}
+
+.loaded {
+  opacity: 0;
+  visibility: hidden;
+}
+
+/* modal parts */
+.modal {
+  padding: 20px;
+}
+
+.modal__wrapper {
+  background-color: #fff;
+  width: 30%;
+  margin: 20px auto;
+  padding: 20px;
+  border: 2px solid black;
+  position: absolute;
+  top: 40px;
+  left: 0;
+  right: 0;
+  z-index: 2;
+}
+
+.modal__request-btn {
+  margin: 20px auto;
+}
+
+.modal__open-btn {
+  margin: 0 auto;
+}
+
+.modal__close-link {
+  color: blue;
+  text-decoration: underline;
+  transition : all 0.3s;
+}
+
+.modal__close-link:hover {
+  color: red;
+  cursor: pointer;
+}
+
+.modal__mask {
+  background-color: rgba(0,0,0,0.4);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 120%;
+  z-index: 1;
+}
+
+#js-lists li:nth-child(n+2){
```

Comment:
`js-lists` should be use only js. use `.lists`

Reply:
I implemented using 'classList.add' to add class name for each of `li`. This time I added not 'ul' but 'li' so I named not 'lists' but 'list'. Is that correct?